README.md

```diff
@@ -27,14 +27,13 @@ This list contains only stage 1 proposals and higher that have not yet been with
 |   | [Optional catch binding][optional-catch]                         | Michael Ficarra                                 | 3 |
 |   | [`function.sent` metaproperty][function-sent]                    | Allen Wirfs-Brock                               | 2 |
 |   | [`String.prototype.{trimStart,trimEnd}`][trims]                  | Sebastian Markbage                              | 2 |
-|   | [Class and Property Decorators][decorators]                      | Yehuda Katz and Brian Terlson                   | 2 |
+|   | [Class and Property Decorators][decorators] ([update](unified-class)) | Yehuda Katz and Brian Terlson                   | 2 |
```

Comment:
This is a broken link. I think it should be `([update][unified-class])`.

Reply:
Thanks, fixed